README.md

```diff
@@ -437,50 +437,68 @@ tabular_explainer = TabularExplainer(clf.steps[-1][1],
 
 ## Visualizations
 
-Load the visualization dashboard in your noteboook to understand and interpret your model:
+Load the visualization dashboard in your notebook to understand and interpret your model:
 
-### Global visualizations
+```python
+from interpret_community.widget import ExplanationDashboard
 
-The following plots provide a global view of the trained model along with its predictions and explanations.
+ExplanationDashboard(global_explanation, model, datasetX=x_test, trueY=y_test)
+```
+Once you load the visualization dashboard, you can investigate different aspects of your dataset and trained model via four tab views: 
 
-|Plot|Description|
-|----|-----------|
-|Data Exploration| Displays an overview of the dataset along with prediction values.|
-|Global Importance|Aggregates feature importance values of individual datapoints to show the model's overall top K (configurable K) important features. Helps understanding of underlying model's overall behavior.|
-|Explanation Exploration|Demonstrates how a feature affects a change in model’s prediction values, or the probability of prediction values. Shows impact of feature interactions.|
-|Summary Importance|Uses indiviual feature importance values across all data points to show the distribution of each feature's impact on the prediction value. Using this diagram, you investigate in what direction the feature values affects the prediction values.
-|
+* Model Performance
+* Data Explorer	
+* Aggregate Feature Importance
+* Individual Feature Importance and What-If	
 
+>[!NOTE]
+> Click on "Open in a new tab" on the top left corner to get a better view of the dashboard in a new tab.
 
-![Visualization Dashboard Global](https://docs.microsoft.com/en-us/azure/machine-learning/service/media/machine-learning-interpretability-explainability/global-charts.png)
 
+You can further create custom cohorts (subgroups of your dataset) to explore the insights across different subgroups (e.g., females vs males). The created cohorts can contain more than one filter (e.g., age < 30 and sex = female) and will be visible from all of the four tabs. The following sections demonstrate the visualization dashboard capabilities on a [classification model trained on employee attrition dataset]((https://github.com/interpretml/interpret-community/blob/master/notebooks/simple-feature-transformations-explain-local.ipynb)). Besides the default cohort (including the whole dataset), there are two additional cohorts created: employees with Age <= 35 and employees with Age > 35.
```

Comment:
Very tiny detail - I would suggest changing "females vs. males" to either "female vs. male" or "women vs. men". Using female/male as nouns bothers some people and tends to be used more frequently in the tech space.